plugin/backend_linux.go

```diff
@@ -228,10 +234,45 @@ func (pm *Manager) Pull(name string, metaHeader http.Header, authConfig *types.A
 }
 
 // List displays the list of plugins and associated metadata.
-func (pm *Manager) List() ([]types.Plugin, error) {
+func (pm *Manager) List(filter string) ([]types.Plugin, error) {
+	pluginFilters, err := filters.FromParam(filter)
+	if err != nil {
+		return nil, err
+	}
+	if err := pluginFilters.Validate(acceptedPluginFilterTags); err != nil {
+		return nil, err
+	}
+
+	enabledOnly := false
+	disabledOnly := false
+	if pluginFilters.Include("enabled") {
+		if pluginFilters.ExactMatch("enabled", "true") || pluginFilters.ExactMatch("enabled", "1") {
```

Comment:
I think we only allow `enable` on boolean filters (like `dangling`). I would expect the same here.